### DISCUSSION TITLE
E621 Link Support

### DISCUSSION THREAD

**Discussion by Thedarklurch** [Official-Husko/NN-Downloader#10] [Ideas]
A link support for e621 would be great.
That you can simply enter a link to the e621 pool and it will then be downloaded.

**Comment by Official-Husko** [comment]
I'm a bit unsure on what exactly you mean so do you have an example link perhaps? 

PS: Please use the issues page next time. Thank you

**Reply by Thedarklurch** [reply]
Example link: https://e621.net/pools/12345
This is what the ones on the left look like and there are individual pictures matched together, whether of a comic or of the same character. 

I wouldn't be sure, that's why I did it this way 😅

**Reply by Official-Husko** [reply]
> Example link: https://e621.net/pools/12345 This is what the ones on the left look like and there are individual pictures matched together, whether of a comic or of the same character.
> 
> I wouldn't be sure, that's why I did it this way 😅

Did you by chance just write 12345 at the end? Also thank you for the example, i will look into it as soon as possible.

**Reply by Thedarklurch** [reply]
I only wrote 12345, there are usually always 5 numbers behind it, can also send you an original link to a pool if that is allowed here.

**Reply by Official-Husko** [reply]
Oh no its alright its just that it was an rather unlucky coincidence on what is posted in said pool. Either way i will take a look when i get time.

**Reply by Thedarklurch** [reply]
Oh lol, I only now see that there is also a pool behind it xD haha

**Comment by Official-Husko** [comment]
@Thedarklurch Turns out this feature is already supported. You can simply download a pool by entering "pool:12345" in the tags for example which will only display and download the images in said pool. Refer to the cheatsheet on the website you want to download from so for example here is [e621](https://e621.net/help/cheatsheet). Unless you are looking for additional features i would consider this complete.
